components/menu/demo/theme.md

```diff
@@ -1,10 +1,18 @@
 ---
 order: 4
-title: 主题
+title: 
+  zh-CN:主题
+  en-US:Theme
 ---
 
+## zh-CN
+
 内建了两套主题 `light|dark`，默认 `light`。
 
+## en-US
+
+Build in two themes `light|dark`, default `light`.
```

Comment:
There are two built-in themes: 'light' and 'dark'. The default value is 'light'.